src/Framework/FieldsAPI/Name.php

```diff
@@ -7,21 +7,4 @@
 class Name extends Group
 {
     const TYPE = 'name';
-
-    public static function make($name)
```

Comment:
I'm not a fan of removing the files from this group. It basically means this group can be anything, and I want to keep it closed by default. I'd rather add functions to make something like this possible:
```php
Name::make('Name')
  ->showHonorific()
  ->firstNameLabel('First Name')
  ->lastNameRequired();
```

It will probably feel tedious, but then we're probably encapsulating this field.

I'll be honest, I'm going back and forth as to how I'd recommend doing this. I don't want to make this overly complex, but I want to be clear that Name is a pre-defined group (you cannot directly modify its child nodes), with limited control over what can be modified within the group. I'm evening imagining an abstract `FixedGroup` framework class that's a child of `Group` which locks down the `append` and such methods, and has an abstract protected function `SetGroupFields` which appends the fields. Maybe on the latter. I'm just brainstorming.

Reply:
I've been toying with a couple different options. Right now I'm leaning towards putting the fields back in the the class and updating the nodes with labels, placeholders, etc. So that class defines what is in the group, while still having access to change labels, etc using the standard Fields API syntax.

Reply:
I've put together a new method, paired with a magic method, that provides access to the child nodes for easy updates. Thoughts?

```php
$node = Name::make('name')->tap(function ($group) use ($block) {
    $group->firstName
        ->label($block->attributes->firstNameLabel)
        ->placeholder($block->attributes->firstNamePlaceholder);
});
```

Reply:
Is this `tap` method for all Node types? Or groups? It's a nice utility!

I'm inclined to have a more specific `Name::make('name')->modifyFirstName(function(Text $firstName) {...})` type of function so folks don't have to know how to search the Group structure (and know its id).